src/components/dock/examples/dock-colors.tsx

```diff
@@ -0,0 +1,101 @@
+import { Component, h, State } from '@stencil/core';
+import { DockItem } from '../dock.types';
+
+/**
+ * Using color properties for individual Dock items
```

Comment:
This one doesn't work at the moment, since I removed the colour variables from the interface. But if there really is a current use case for them, we should of course add them back in. If there isn't, I would argue that we should add the support when we have a use case. But I'm not ardent about it 😄